DAS/das/V3_fake_module.py

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Is this in here on purpose or is it a remnant from a failed merge at some point?